stock_picking_operation_quick_change/tests/test_stock_picking_operation_quick_change.py

```diff
@@ -1,10 +1,12 @@
-# Copyright 2017 Tecnativa - Sergio Teruel
+# Copyright 2022 Tecnativa - Sergio Teruel
```

Comment:
Please, do not change copyright years. When only 1 year is present, is intended as "from 2017".